scene/main/viewport.cpp

```diff
@@ -1264,6 +1264,10 @@ Camera2D *Viewport::get_camera_2d() const {
 	return camera_2d;
 }
 
+Vector2 Viewport::get_stretch_scale_factor() const {
+	return stretch_transform.get_scale();
+}
```

Comment:
I'm not sure about this, considering I expect this value to be used relatively often. If users have to get the right basis out to get the scale factor, this would reduce its usability.

Reply:
Done.